lego/apps/articles/serializers.py

```diff
@@ -33,6 +33,7 @@ class Meta:
             "content",
             "created_at",
             "pinned",
+            "youtube_url",
```

Comment:
Correct me if I'm wrong, but this only adds the url when viewing a specific article right? What happens when a youtube article is on the frontpage? Or on the `/articles`-list view?

Reply:
Then the normal cover photo will be present 👍